modules/templates/helper.go

```diff
@@ -522,9 +522,15 @@ func parseOther(defaultSize int, defaultClass string, others ...interface{}) (in
 	return size, class
 }
 
+func avatarHtml(src string, size int, class string, name string) template.HTML {
+	sizeStr := fmt.Sprintf(`%d`, size)
+
+	return template.HTML(`<img class="` + class + `" src="` + html.EscapeString(src) + `" alt="` + html.EscapeString(name) + `" width="` + sizeStr + `" height="` + sizeStr + `"/>`)
```

Comment:
This does escape the`&` in the avatar `src` in an effort to prevent HTML injection. I don't think it will break our own or gravatar URLs but it' something to watch out for for future avatar sources.

Reply:
Actually, decided against this escaping as I don't think it's really possible that a user controls the avatar URLs, they are based on hashes in all cases.